### DISCUSSION TITLE
本科毕业论文，请教下表格索引怎么关闭&achievements的编译问题

### DISCUSSION THREAD

**Discussion by hearhh** [whutug/whu-thesis#207] [Q&A] [answered]
我的论文没有包含表格，但我没有找到控制表格索引的代码，所以编译结果里面有一个孤零零的“表格索引”，想问问怎么关...
btw还想问下achievements为什么编译不出来呢？

**Comment by sikouhjw** [comment, marked as answer]
> 我的论文没有包含表格，但我没有找到控制表格索引的代码，所以编译结果里面有一个孤零零的“表格索引”，想问问怎么关...

https://github.com/whutug/whu-thesis/blob/6f19c2e0d34bbfc752f86a7ea505154a185ff595/whu-thesis-demo.tex#L24-L29

> btw还想问下achievements为什么编译不出来呢？

如果 Overleaf 可以，但你不行，就应该反思一下

**Reply by hearhh** [reply]
翻到这里了愣是没看见hhh，感谢解答
